native/helpers/contributors.js

```diff
@@ -98,4 +98,8 @@ export const contributors = new Map([
     'DhanushNehru',
     { name: 'Dhanush Nehru', pic: 'https://avatars.githubusercontent.com/u/22955675?v=4'}
   ],
+  [
+    'ayomidehassan',
+    { name: 'Ayomide Hassan', pic: 'https://avatars.githubusercontent.com/u/111683116?s=400&u=221fa8f0cdc1c0101bc5950a73e6e888dfb17b0a&v=4' },
+  ],
```

Comment:
The username that you add in `contributors.js` should be same as the `developer` value in your `challenges.js`.

Otherwise your avatar won't appear.

![image](https://github.com/sadanandpai/frontend-mini-challenges/assets/82361490/eff692c9-b400-47ab-8aa9-34ea6ce31e45)

Reply:
done sir